### ISSUE TITLE
deluge-2.0.3 missing runtime dependencies

### ISSUE THREAD

**Issue by xtraeme** [void-linux/void-packages#12811]
void-666 xbps-9999

deluge (as well as -gtk, -web) fail to run due to missing dependencies.

The packages I had to install manually are the required ones.

```
[juan@leysa void-packages]$ xbps-query -s deluge
[*] deluge-2.0.3_1     Fully-featured cross-platform BitTorrent client
[*] deluge-gtk-2.0.3_1 Fully-featured cross-platform BitTorrent client - GTK+ frontend
[juan@leysa void-packages]$ xbps-query -m|grep python3
python3-Mako-1.0.12_1
python3-Pillow-6.0.0_1
python3-rencode-1.0.6_2
python3-setproctitle-1.1.10_5
[juan@leysa void-packages]$
```

**Comment by xtraeme** [comment]
cc @mobinmob

**Comment by xtraeme** [comment]
https://github.com/deluge-torrent/deluge/blob/develop/requirements.txt

So weird that nobody has added a new hook to xbps-src that checks python's `requirements.txt` file...

**Comment by xtraeme** [comment]
Or we could look at `setup.py` instead:

- for building: https://github.com/deluge-torrent/deluge/blob/develop/setup.py#L537
- for running: https://github.com/deluge-torrent/deluge/blob/develop/setup.py#L538

**Comment by mobinmob** [comment]
It did not crop up during testing (I already have the deps in my system).
pillow and setproctitle are listed as optional [here](https://dev.deluge-torrent.org/wiki/Installing/Source), mako is listed as a dep for deluge-web in the template...
I will test by removing the deps from my system. A hook will be very nice indeed.

**Comment by maciozo** [comment]
Can confirm I had to install those deps as well.

**Comment by SolitudeSF** [comment]
probably can be closed

**Comment by jnbr** [comment]
fixed in 4f20e40d11aba5c806892d291d16ec6587a2ad9a
